Sources/Clappr_iOS/Classes/Base/Playback.swift

```diff
@@ -127,7 +127,7 @@ extension Playback {
         return 0
     }
 
-    @objc var usingDVR: Bool {
+    @objc open var usingDVR: Bool {
```

Comment:
Hey @leandroalonso and @dezouzart , are you using this property in another module? If the event is triggered only when a change in dvr status is observed, there is no need to pass `usingDVR` in userInfo. I suggest change the name of the event to something like `didChangeDvrStatus`.

Reply:
Hey guys, I'm going to approve this PR. As we discussed before, those changes will be made in another PR.